### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by Phionamary**
### What does this PR do?
This PR merges the changes on the feature branch into  gh-pages

**Description of the tasks to be completed**
* Set up endpoints for the API that accomplish the following:
  * Place a new order
  * Get all orders
  * Get a particular order
  * Update order status
  * Delete an order

**What are the relevant pivotal tracker stories?**
   160547735

### CODE REVIEW

File: rest_api/views.py [hunk 1]
```diff
@@ -13,8 +13,8 @@ def _get_order(Request_ID):
     return [order for order in orders if order['Request_ID'] == Request_ID]
 
 
-def _record_exists(Client_Name):
-    return [order for order in orders if order["Client_Name"] == Client_Name]
+def _record_exists(Request_ID):
+    return [order for order in orders if order["Request_ID"] == Request_ID]
```
Comment (kdaglas) [inline rest_api/views.py]: this is great structuring... it is surely reusable... my only issue is your naming... for some reason it confused me at first then i started getting it but all is now well... maybe next time try not to use same words for different values... or let the keys have some name relating to the values... i hope am making sense... but all is great.

**Comment by kdaglas** [comment]
Your ReadME still has some conflicting changes... maybe work that out and push again

**Comment by Phionamary** [comment]
@kdaglas I have resolved the conflict....

**Comment by ramzinc** [comment]
Brilliant Use Of Classes had not even considered them.

**Comment by kdaglas** [comment]
@Phionamary Dope now your readME is ok... great stuff... and your OOP is on point

**Comment by Phionamary** [comment]
@ramzinc  Thank you.

**Comment by Phionamary** [comment]
@billkabanga Working on it...

**Comment by ekumamait** [comment]
Maybe you could include that .gitignore file among the files you’re ignoring so your repo looks exquisite otherwise 👍🏾

**Review by Kasulejoseph — COMMENTED** [review]
your work is pretty good. but, i would recommend u to improve on the file structure. its not a good practise to run the flask app directly.. imagine your going to add more features in the app how will they be accessed if the running the app directly from views

File: rest_api/models.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
+orders = []
+
+class Orders(object):
```
Comment (mozzy22) [inline rest_api/models.py:3]: u may not need to inheirt from object , its a default in python 3

File: rest_api/models.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+orders.append(order3.to_json())
+
+order4 = Orders(4, "Rosemary Nakitte", "KFC Bukoto", "A large bucket of chicken wings, family size fries and a large coke", 2, "10/09/2018", "Approved")
+orders.append(order4.to_json())
+
+order5 = Orders(5, "Nassanga Ann", "Pizza Bond", "Large steak pizza. Lots of toppings", 5, "10/10/2018", "Pending")
+orders.append(order5.to_json())
+
+order6 = Orders(6, "Phiona Mary Kigai",	"KFC Bukoto", "A large bucket of chicken wings, family size fries and a large coke", 1, "10/09/2018", "Pending")
+orders.append(order6.to_json())
```
Comment (mozzy22) [inline rest_api/models.py:34]: u dont need to use dummy data inside your code , u can add it via post man

File: rest_api/views.py [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+        abort(400)'''
+
+    Actions = request.json.get('Actions')
+    
+    order = {'Request_ID': order_id,'Client_Name': Client_Name,'Restaurant': Restaurant, 'Detail':Detail, 'Quantity': Quantity, 'Date':Date, 'Actions': Actions}
+    orders.append(order)
+    return jsonify({'order': order}), 201
+
+
+@app.route('/api/v1.0/orders/<int:Request_ID>', methods=['PUT'])
```
Comment (Kasulejoseph) [inline rest_api/views.py]: hey had forgotten this one, your using a versioning  that is different from the one we were told to use `/api/v1/` not `/api/v1.10/`

Comment (Phionamary) [reply rest_api/views.py]: Worked on it. Thank you for the reminder

**Pull request merged**